torch/onnx/symbolic_helper.py

```diff
@@ -439,6 +439,12 @@ def _if_scalar_type_as(self, tensor):
     return self
 
 
+@_beartype.beartype
+def _is_custom_domain(node: _C.Node) -> bool:
+    domain = node.kind().split("::")[0]
+    return domain not in {"onnx", "aten", "prim"}
```

Comment:
We should move away from using private methods outside of their own modules. In this case we can add it to jit_utils. I would also consider decoupling string stuff from Nodes. I will take a closer look later. Happy to discuss

Reply:
Agree and will move it. I actually thought about adding this into jit_utils.py, but then I saw lots of node processing and bool function in symbolic_helper.py, so I went with it.